src/Microsoft.FileFormats/ELF/ELFProgramHeader.cs

```diff
@@ -20,6 +15,17 @@ public enum ELFProgramHeaderType : uint
         Phdr = 6
     }
 
+    [Flags]
+    public enum ELFProgramHeaderFlags : uint
```

Comment:
Is this used anywhere at all?

Reply:
This will be used in some upcoming diagnostics extensions work. I wanted to get them in now.